src/ttsnake.c

```diff
@@ -229,6 +234,16 @@ void draw (scene_t* scene, int number)
   if (number == GAME_OVER) { /* if displayed scene is the game over scene */
     /* we must show the final score */
     mvwprintw(main_window, NROWS*3/4, NCOLS/2-7, "Score: %d", block_count);
+    
+    /*Instructions to enter nickname*/
+    if(!entered_score) {
+      mvwprintw(main_window, NROWS*3/4 + 1, NCOLS/5-7, "Type a nick of at most 4 letters to save your score.");
```

Comment:
I think that we should format the instructions as follows:
```suggestion
      mvwprintw(main_window, NROWS*3/4 + 3, NCOLS/5 - 2, "Type a nick of at most 4 letters to save your score.");
```